backend/config/settings/base.py

```diff
@@ -399,3 +399,10 @@ def traces_sampler(sampling_context):
 
 # This string is prefixed to the beginning of every email (subject).
 EMAIL_SUBJECT_PREFIX = f"[IndEAA {APP_ENV}] "
+
+###################
+# Use HTTPS headers
+###################
+
```

Comment:
commnt on the reason why this is needed .... one sentence and reference to a docs of stackoverflow post

Reply:
oauth requires https for urls, however internally http is used so we need to use the X-forwarded header to ensure the uri gets generated with https to prevent uri mismatch due to http url getting generated
https://stackoverflow.com/questions/62047354/build-absolute-uri-with-https-behind-reverse-proxy